resolwe_bio/processes/slamdunk/alleyoop_utrrates.py

```diff
@@ -0,0 +1,70 @@
+"""Run Alleyoop utrrates tool on Slamdunk results."""
+import os
+
+from plumbum import TEE
+
+from resolwe.process import Cmd, DataField, FileField, Process, StringField
+
+
+class AlleyoopUtrRates(Process):
+    """Run Alleyoop utrrates."""
+
+    slug = 'alleyoop-utr-rates'
+    process_type = 'data:alleyoop:utrrates'
+    name = 'Alleyoop UTR Rates'
+    requirements = {
+        'expression-engine': 'jinja',
+        'executor': {
+            'docker': {
+                'image': 'resolwebio/slamdunk:1.0.0'
+            },
+        },
+        'resources': {
+            'cores': 1,
+            'memory': 16384,
+        },
+    }
+    entity = {
+        'type': 'sample',
+    }
+    data_name = '{{ slamdunk|sample_name|default("?") }}'
+    version = '1.0.0'
+
+    class Input:
+        """Input fields for AlleyoopUtrRates."""
+
+        ref_seq = DataField('seq:nucleotide', label='FASTA file containig sequences for alingnig')
+        regions = DataField('bed', label='BED file with coordinates of regions of interest')
+        slamdunk = DataField('alignment:bam:slamdunk', label='Slamdunk results')
+
+    class Output:
+        """Output fields to process AlleyoopUtrRates."""
+
+        report = FileField(label='Tab-separated file containing conversion rates on each region of interest')
+        plot = FileField(label='Region of interest conversion rate plot file')
```

Comment:
Could we live without the "file" part, i.e."Region of interest conversion rate plot"?